### DISCUSSION TITLE
[Project Beta] Dropdowns should remember the most recent selections

### DISCUSSION THREAD

**Discussion by jeffbax** [community/community#12411] [Projects and Issues] [answered]
Running into a really annoying behavior that just slows everything down. But whenever I'm using dropdowns for either converting a note into an issue (and picking a Repo) the immediate loaded items never features our # 1 most active repository, so I have to go and slowly search and wait for the API call to finish.

Similarly, when picking milestones in the list... I've been picking the same one over and over, but its always showing some random sort of them rather than the ones I'm using pretty exclusively within the context of a project.

It seems like currently the Repos are just... randomly chosen, and Milestones are sorted alphabetically (special characters at the bottom)

**Comment by ghost** [comment]
> Running into a really annoying behavior that just slows everything down. But whenever I'm using dropdowns for either converting a note into an issue (and picking a Repo) the immediate loaded items never features our # 1 most active repository, so I have to go and slowly search and wait for the API call to finish.
> 
> Similarly, when picking milestones in the list... I've been picking the same one over and over, but its always showing some random sort of them rather than the ones I'm using pretty exclusively within the context of a project.
> 
> It seems like currently the Repos are just... randomly chosen, and Milestones are sorted alphabetically (special characters at the bottom)

**Comment by jeffbax** [comment, marked as answer]
Seems like something was improved on this front, this can be closed 😃
